.travis.yml

```diff
@@ -1,15 +1,12 @@
 sudo: false
-cache: false
+cache: yarn
 language: node_js
 node_js:
   - 6.9.2
-install:
-  - npm install
-before_script:
```

Comment:
Why has `before_script` section disappeared?

Reply:
This section is not used anymore. The `secrets.json` file is not used here and I have added `grunt-cli` to the devDependencies.